compiler/x/codegen/X86Ops.hpp

```diff
@@ -1217,6 +1217,102 @@ typedef enum
    IA32EFlags_CF = 0x10
    } TR_EFlags;
 
+namespace TR
+   {
+   namespace X86
+      {
+      namespace Instruction
```

Comment:
In my mind `X86::`  is an acceptable subclassification under `TR::`, but I feel chopping it up even finer under `Instruction::` is providing little additional isolation for this particular feature.  I prefer the `Instruction::` namespace removed.